core/trino-main/src/main/java/io/trino/operator/aggregation/partial/PartialAggregationController.java

```diff
@@ -47,27 +57,25 @@ public boolean isPartialAggregationDisabled()
         return partialAggregationDisabled;
     }
 
-    public synchronized void onFlush(long rowsProcessed, long uniqueRowsProduced)
+    public synchronized void onFlush(long bytesProcessed, long rowsProcessed, long uniqueRowsProduced)
     {
-        if (partialAggregationDisabled) {
-            return;
-        }
-
+        totalBytesProcessed += bytesProcessed;
```

Comment:
why do we count input bytes and not output bytes?

Reply:
discussed offline. It's not a bit difference one way or the other.